virtcontainers/acrn.go

```diff
@@ -737,20 +743,14 @@ func (a *Acrn) GetACRNUUIDBytes(uid string) (uuid.UUID, error) {
 
 // GetNextAvailableUUID returns next available UUID VM creation
 // If no validl UUIDs are available it returns err.
-func (a *Acrn) GetNextAvailableUUID(uuidstore *store.VCStore) (string, error) {
+func (a *Acrn) GetNextAvailableUUID() (string, error) {
```

Comment:
Not part of this PR, but there is a typo in the function header comment ("validl" => "valid").